charts/kubeclarity/values.yaml

```diff
@@ -341,3 +336,15 @@ kubeclarity-postgresql:
 
 # End of KubeClarity Postgres Values
 #######################################################################################
+
+#######################################################################################
+## KubeClarity Postgres Authentication Values
+kubeclarity-postgresql-auth:
+  # default values can be used if kubeclarity-postgresql is used for pgsql deployment
+  existingSecret: kubeclarity-postgresql-secret
+  database: kubeclarity
+  user: postgres
+  #host: override host if needed
```

Comment:
can you move the comment above the field just to make it more obvious, and also add a bit more documentation around it use:
```
# If kubeclarity-postgresql is not enabled then you must provide
# the host of a remote postgresql server.
# host:
```

Reply:
I added a `kubeclarity-postgresql-external` key to avoid confusion.